src/main/resources/assets/restrictedportals/lang/ru_ru.json

```diff
@@ -0,0 +1,6 @@
+{
+  "achievement.rpunlock.unlock": "[dim] Разблокирован!",
+  "achievement.rpunlock.desc": "Получить [item]",
+  "achievement.rpunlock.message": "Пожалуйста, переведено сообществом модовестник получите [item], чтобы войти [dim]",
```

Comment:
"Please, *translated by modovestrnik community* get [item] to enter [dim]"
Advertisement of his VK community, just like in any other of his "translations"